### PR TITLE
hotfix-frontend-bug feature flag

### PR DESCRIPTION
**Pull request by jerryfreshworks**
**Resolves:**

- fix the frontend routing bug

**Changes:**

- adding flag to the route

**References:**

- [CSBISS-78](https://justice.gov.bc.ca/jira/browse/CSBISS-78)

### Prerequisites

- [x] My code follows the code style of this project. (https://sites.google.com/freshworks.io/virtual-clinic/technical-overview/coding-guidelines-dev-team-agreement and https://sites.google.com/freshworks.io/virtual-clinic/technical-overview/coding-guidelines-dev-team-agreement/full-stack)
- [x] I have considered the maintainability of the code I have written
- [ ] I have updated the documentation according to my changes.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -35,14 +38,14 @@ const App = () => {
...
-                <Route exact path="/bookings" component={Booking} />
+                <WithFlagRoute exact path="/bookings" component={Booking} />
                 <Route exact path="/create" component={CreateBooking} />
                 <Route
                   exact
                   path="/directory"
                   component={InterpreterDirectory}
                 />
-                <Redirect to="/bookings" />
+                <Redirect to={checkIfFlag() ? '/create' : '/bookings'} />
```
Comment (fwpushan) [inline frontend/src/App.tsx:48]: I think /directory will be more appropriate

**Review by fwpushan — COMMENTED** [review]
Only one comment, please get clarification from Graeme

File: frontend/src/App.tsx (hunk 1 above)
Comment (jerryfreshworks) [reply frontend/src/App.tsx:48]: image

**Comment by jerryfreshworks** [comment]
based on Graeme's feedback. "Searching Directory" is the default page. So I will merge this PR.

File: frontend/src/App.tsx (hunk 1 above)
Comment (jerryfreshworks) [reply frontend/src/App.tsx:48]: @fwpushan  And thanks for good question!

**Pull request merged**